### PR TITLE
Update application-manifests.md

### PR DESCRIPTION
**Pull request by asklar**

### CODE REVIEW

**Comment by PRMerger14** [comment]
@asklar : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by asklar** [comment]
CC @oldnewthing @jonwis

**Comment by jonwis** [comment]
See also https://docs.microsoft.com/en-us/dotnet/framework/configure-apps/file-schema/runtime/assemblyidentity-element-for-runtime which is the .NET equivalent documentation.

**Comment by asklar** [comment]
@jonwis looks like the .NET docs are missing the ARM* ones (they link to the doc for the enum which has the right values but they're missing in the main page)

File: desktop-src/SbsCs/application-manifests.md [hunk 1]
```diff
@@ -111,7 +111,7 @@ The **assemblyIdentity** element has the following attributes. It has no subelem
...
-| **processorArchitecture** | Specifies the processor. The valid values are x86 for 32-bit Windows and ia64 for 64-bit Windows. Optional.                                                                                                                                                                                                                                                                                                                       |
+| **processorArchitecture** | Specifies the processor. The valid values are `x86`, `amd64`, `arm` and `arm64`. Optional.                                                                                                                                                                                                                                                                                                                       |
```
Comment (oldnewthing) [inline desktop-src/SbsCs/application-manifests.md]: Maybe we should say

> Valid values include `x86`, `amd64`, `arm`, and `arm64`.

That way, this documentation doesn't become instantly *wrong* every time we onboard a new architecture. (It just becomes instantly *suboptimal*, which is better than being instantly wrong.)

Comment (asklar) [inline desktop-src/SbsCs/application-manifests.md]:
```suggestion
| **processorArchitecture** | Specifies the processor. Valid values include `x86`, `amd64`, `arm` and `arm64`. Optional.                                                                                                                                                                                                                                                                                                                       |
```

**Comment by PRMerger6** [comment]
@asklar : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Review by mcleanbyron — APPROVED** [review]

**Pull request merged**